src/js/translate-visitor.ts

```diff
@@ -19,9 +19,13 @@ export default class TranslateVisitor extends types.PathVisitor implements  type
     };
     currentContext: types.Context;
 
-    constructor(private loader: TranslateLoaderContext) {
+    constructor(private loader: TranslateLoaderContext, options: any) {
         super();
 
+        options = options || {};
```

Comment:
You can use `Object.assign` here to merge the user options with our default options.

```js
this.options = Object.assign({}, options, this.options);
```